RestClient.cpp

```diff
@@ -2,32 +2,23 @@
 
 #ifdef HTTP_DEBUG
 #define HTTP_DEBUG_PRINT(string) (Serial.print(string))
-#endif
-
-#ifndef HTTP_DEBUG
+#else
 #define HTTP_DEBUG_PRINT(string)
 #endif
 
 RestClient::RestClient(const char* _host){
     host = _host;
     port = 80;
-    ssl = 0;
-    fingerprint = NULL;
-    num_headers = 0;
-    if (!contentType) {
-        contentType = "application/x-www-form-urlencoded";  // default
-    }
+    ssl = false;
+    contentType = "application/x-www-form-urlencoded";
+	rawLastResponse = "";
 }
 
 RestClient::RestClient(const char* _host, int _port){
     host = _host;
     port = _port;
-    ssl = 0;
-    fingerprint = NULL;
-    num_headers = 0;
-    if (!contentType) {
-       contentType = "application/x-www-form-urlencoded";  // default
-    }
+    ssl = false;
+    contentType = "application/x-www-form-urlencoded";
```

Comment:
The consumer may still call `setContentType` so we should keep the conditional logic around these
```
    if (!contentType) {	
        contentType = "application/x-www-form-urlencoded";  // default	
    }
```